deploy/kubernetes/charts/orders/values.yaml

```diff
@@ -139,6 +148,15 @@ rabbitmq:
 
   affinity: {}
 
+  persistentVolume:
+    enabled: false
+    annotations: {}
+    labels: {}
+    accessModes:
+    - ReadWriteOnce
+    size: 10Gi
+    storageClass: gp2
```

Comment:
I believe that the convention is to set this to an empty string so that it uses the default storage class of the cluster.